server/py_files/models/SentenceEmbeddings/SentenceEmbeddings.py

```diff
@@ -104,15 +104,15 @@ def group_similarity_score(self, groups_of_sents, method='custom'):
                         sent_vec = self.vector(sent)
                         sent_l2 = math.sqrt(np.dot(sent_vec, sent_vec))
                         score = np.dot(fromsent_vec, sent_vec) / (fromsent_l2 * sent_l2)
-                        score = (score + 1)/2
-                        # score = 1 if score > 1 else (0 if score < 0 else score)
+                        # score = (score + 1)/2
```

Comment:
@astrebeck : the `gensim` method is giving substantially different results than the `custom` method on the demo_resume (which I have been using in all the demos) - IF we use the old score_capping way of forcing it b/w (0, 1)

I think the issue with the `very neutral` scores we saw with your resume - was with the new score_capping way => the `(score + 1)/2` is kind of smoothening out any kind of real standout differences

I have changed it back - I think its a better representation of what we want - I ll show u once u up, we can test on your resume as well